core/kernel/console.c

```diff
@@ -96,9 +96,12 @@ void configure_console_from_dt(void)
 		return;
 	}
 
-	stdout_data = strdup(prop->data);
+	/* We can't use strdup() because it tries to use malloc() */
+	stdout_data = kmalloc(strlen(prop->data) + 1);
```

Comment:
I'd prefer a `kstrdup()`.

Reply:
Any suggestion, about where it should be implemented?

e.g. is "isoc/strdup.c" the right place?

Reply:
No, it should be "ext/kstrdup.c", with prototype in `<string_ext.h>`

I'm not a big fan of this split of our "libc", but I'm afraid that's how it still is.

Reply:
Added "kstrdup()" function and reworked patch to use it.